packages/boost-libs/build

```diff
@@ -0,0 +1,25 @@
+#!/bin/bash
```

Comment:
@nickbp Can you clarify here?

@cmaloney This was copied out of our enterprise distro to OSS the project, I'm not sure what the right answer is on this.